fabric-item-api-v1/src/testmod/java/net/fabricmc/fabric/test/item/CustomDamageTest.java

```diff
@@ -79,5 +103,15 @@ public ItemStack getRecipeRemainder(ItemStack stack) {
 
 			return ItemStack.EMPTY;
 		}
+
+		@Override
+		public ItemEnchantmentsComponent getIntrinsicEnchantments(ItemStack stack) {
+			if (stack.isDamaged()) {
+				System.out.println("Damaged, adding silk touch");
```

Comment:
I think it should be `player.sendMessage` or `LOGGER.log` :D

Reply:
Oop, that's a debug message I forgot to remove